apps/go/manager/types/circular_buffer.go

```diff
@@ -27,7 +28,30 @@ type CircularBuffer struct {
 // Gets the sample index given a step direction (positive: 1 or negative: -1) and for a given marker (start or end of buffer)
 func (buffer *CircularBuffer) StepIndex(step uint32, marker string, positive_step bool, l *zerolog.Logger) error {
 
-	l.Debug().Int("buffer.Indexes.End", int(buffer.Indexes.Start)).Int("buffer.Indexes.End", int(buffer.Indexes.End)).Msg("Circular indexes.")
+	l.Debug().Int("buffer.Indexes.Start", int(buffer.Indexes.Start)).Int("buffer.Indexes.End", int(buffer.Indexes.End)).Int("step", int(step)).Msg("Circular indexes moving.")
```

Comment:
```suggestion
	l.Debug().
	  Int("buffer.Indexes.Start", int(buffer.Indexes.Start)).
	  Int("buffer.Indexes.End", int(buffer.Indexes.End)).
	  Int("step", int(step)).
	  Msg("Circular indexes moving.")
```
Always avoid long lines and break on each param or set of params to keep it easier to read.